src/rebar3_hex_publish.erl

```diff
@@ -419,17 +419,17 @@ is_empty_prop(K, PropList) ->
           false
     end.
 
-is_unstable_dep({_, {pkg, Pkg, Ver, _, _}, _}, Acc) -> 
+is_unstable_dep({_, {pkg, Pkg, Ver, _, _}, _}, Acc) ->
```

Comment:
Unwarranted white space deletions. Let me know if I should revert or create a separate commit.